composer.json

```diff
@@ -13,7 +13,8 @@
         }
     },
     "require": {
-        "php": "^5.6 || ^7.0"
+        "php": "^5.6 || ^7.0",
+        "scriptfusion/static-class": "^1"
```

Comment:
Yes, but that's not going to happen due to two reasons:

 * it creates a dependency for all packages relying on stdlib ([ref](https://packagist.org/packages/zendframework/zend-stdlib/dependents)
 * use-case scenario is tiny: 2LOC (vs 1LOC) per class in this package
 * it is not a package in ZF, therefore you have to have a stability that is either equivalent or better than ZF's in order to include it